src/pages/Files.vue

```diff
@@ -72,6 +72,7 @@ export default {
           <oc-avatar
             class="oc-topbar-personal-avatar"
             src="https://picsum.photos/64/64?image=1027"
```

Comment:
It's here @DeepDiver1975 It was missing in OCAvatar at the one with placeholder but that is fixed already and pushed.

Reply:
The point is: if src is not *yet* set we will get this warning in phoenix as well. So it should be marked as required: false

Reply:
Done and the test passes so I guess it should be fine now.